app/helpers/users/product_colors_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Users::ProductColorsHelper
+  def load_avatar product_color_id
+    product_color = ProductColor.by_id product_color_id
+    product_color.first.product_avatar
+  end
+
+  def productname_product_color product_color_id
+    (ProductColor.by_id product_color_id).first.product_name
+  end
+
+  def colorname_product_color product_color_id
+    (ProductColor.by_id product_color_id).first.color_color
```

Comment:
sao e để ngoặc ngược đời thế
sao k để là 
```suggestion
    ProductColor.by_id(product_color_id).first.color_color
```

Reply:
thế này mỗi vòng lặp phải find product color 2 lần à ?